README.md

```diff
@@ -125,18 +125,21 @@ If the `uri` is not specified in the `parameterSource` section, then it will def
 Eunomia uses the [yq command](http://mikefarah.github.io/yq/) to merge all yaml files in the specified folder. You have to be careful, if you have the same variable name in multiple files. Dictionaries will merge, lists will get overwritten. 
 
 #### Variable Hierarchy
-You can provide a file `hiera.lst`, to allow a variable hierarchy. This will allow for example to specify a default value and overwrite it on an environment level if necessary. This will greatly simplify your configuration and allows for deduplication of data, making your operational life a lot easier.
+You can provide a file `hierarchy.lst`, to allow a variable hierarchy. This will allow you to specify a default value and overwrite it on an environment level if necessary. This will greatly simplify your configuration and allows for deduplication of data, making your operational life a lot easier.
 
 The contents of the file are simply relative path names, with the base being `contextDir`.
 
-Example `hiera.lst`:
+Example `hierarchy.lst`:
 ```
-../defaults #this is first
-../marketing #this is second
-../development #this is third
-# lastly the directory containing hiera.lst is merged
+../defaults    #this is first ... lowest priority
+../marketing   #this is second
+../development #this is third ... highest priority
 ```
-In this case it will load all yaml files from `../defaults`, then merge it with everything in `../marketing`, merge it with everything in `../development`, and lastly merge it with everything defined in `contextDir`.
+
+In this case it will load all yaml files from `../defaults`, then merge it with everything in `../marketing`, and lastly merges it with everything in `../development`. You can also use the relative path `./`, which means it'll also load the variables defined in `contextDir` directly (same folder that as `hierarchy.lst`).
+
+#### Upcoming features
+Once [issue #4](https://github.com/KohlsTechnology/eunomia/issues/4) is resolved, you will be able to specify variable names to dynamically determine the correct folder. This will allow you to only have one `hierarchy.lst`.
```

Comment:
Suggested addition: *"(Technically, it is actually already possible to use environment variables, but without #4, there are just none set that would be of any practical use in `hierarchy.lst`.)"*